config/helm/service-account-client-id.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: capz-manager
+  namespace: capz-system
+  annotations:
+    azure.workload.identity/client-id: "{{.Values.provider.clientId}}"
```

Comment:
Is this the same clientID we use in mc-bootstrap?

Reply:
Just a nit, but please add a newline at EOF. And you can safely omit the `---` separator. 🙂

Reply:
Thanks, I always forget about this 🙈

Reply:
To answer myself `Is this the same clientID we use in mc-bootstrap?`

At least in `glean` it is.